targets/unity-v2/templates/PlayFabSettings.cs.ejs

```diff
@@ -41,7 +41,7 @@ namespace PlayFab
             var settingsList = Resources.LoadAll<PlayFabSharedSettings>("PlayFabSharedSettings");
             if (settingsList.Length != 1)
             {
-                throw new Exception("The number of PlayFabSharedSettings objects should be 1: " + settingsList.Length);
+                throw new MultipleSharedSettingsException(settingsList.Length);
```

Comment:
Yeah, here it is.  Please undo this one, because it'll conflict with another change we're making.

Reply:
Done!